WordPress/gutenberg
# Interactivity API: Remove `data-wp-slot` and  `data-wp-fill`

## What?

Remove not used `data-wp-slot` and `data-wp-fill` directives.

## Why?

To make the Interactivity API public with the minimum amount of directives needed. After 6.5 and getting some feedback, they can be reviewed.

## Testing Instructions




- Check that Search, Navigation, File, Query without force reload and Lightbox features are working.
